astropy/modeling/tests/test_functional_models.py

```diff
@@ -87,18 +87,18 @@ def test_Gaussian2DRotation():
     assert_allclose(value1, value2)
 
 
-def test_Redshift():
+def test_RedshiftWavelength():
     """Like ``test_ScaleModel()``."""
 
     # Scale by a scalar
-    m = models.Redshift(0.4)
+    m = models.RedshiftWavelength(0.4)
     assert m(0) == 0
     assert_array_equal(m([1, 2]), [1.4, 2.8])
 
     assert_allclose(m.inverse(m([1, 2])), [1, 2])
 
     # Scale by a list
-    m = models.Redshift([-0.5, 0, 0.5], model_set_axis=0)
+    m = models.RedshiftWavelength([-0.5, 0, 0.5], model_set_axis=0)
```

Comment:
Huh. Turns out I'm wrong.  This test is fine as is and in fact *only* works because apparently if you specify `model_set_axis` when creating the model the `n_models` is inferred automatically from the length of the relevant axes in the parameters.  i.e. because `model_set_axis=0` it looks at the input [-0.5, 0, 0.5] and infers that this represents a set of three models with different values for `z`.  That's by design too; I'd just forgotten.  All the more reason this could be documented more clearly, I think...  Sorry for the confusion.